Modules/Tokens/src/Tokens.Application/Tokens/Queries/GetToken/Validator.cs

```diff
@@ -9,5 +10,6 @@ public class Validator : AbstractValidator<GetTokenQuery>
     public Validator()
     {
         RuleFor(x => x.Id).ValidId<GetTokenQuery, TokenId>();
+        RuleFor(x => x.Password).NumberOfBytes(1, Token.MAX_PASSWORD_LENGTH).When(x => x.Password != null);
```

Comment:
Here we use `When(x => x.Password != null)`. But in the validator of the `CreateToken` request, for example, we don't use this.
=> I guess the `NumberOfBytes` method can handle null and we don't need the `When` here.

I already removed it.